src/dayview.ts

```diff
@@ -841,23 +597,121 @@ export class DayViewComponent implements ICalendarComponent, OnInit, OnChanges,
                         eventSet.push(displayEvent);
                         rows[startIndex].events = eventSet;
                     }
+
+                    // setup eventsGroupedByCategory
```

Comment:
only do this when dayviewShowCategoryView is true

Reply:
fixed